### PR TITLE
PR2: Home tasks for lesson4 - JS Basics

### PR DESCRIPTION
**Pull request by Vitalina1994**

### CODE REVIEW

**Comment by sttaran** [comment]
node modules should be excluded from commits, add it to gitignore

File: js_basics/task5.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+let circleArea = Math.PI * Math.pow(radius, 2)
+console.log('Area of a circle equals ' + circleArea.toFixed(2))
+
+let length = 18.3
+let width = 34.2
+let rectangleArea = length * width
+console.log('Area of rectangle equals ' + rectangleArea.toFixed(2))
+
+let height = 40
+let cylinderArea = Math.PI * Math.pow(radius, 2) * height
```
Comment (sttaran) [inline js_basics/task5.js:11]: not area but volume

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics/task5.js (hunk 1 above)
Comment (Vitalina1994) [reply js_basics/task5.js]: Resolved

File: .gitignore
Comment (sttaran) [inline .gitignore:1]: Everything in this folder is service files for your IDE. You should not touch these files. Instead you need to create your own `.gitignore` file on the root of project

**Review by sttaran — CHANGES_REQUESTED** [review]

**Pull request merged**